olclient/schemata/edition.schema.json

```diff
@@ -171,38 +163,14 @@
         "additionalProperties": false,
         "required": [ "key" ],
         "properties": {
-          "key": { "$ref": "#/definitions/work_key" }
+          "key": { "$ref": "shared_definitions.json#work_key" }
         }
       }
     },
-    "author_key": {
-      "type": "string",
-      "pattern": "^/authors/OL[0-9]+A$"
-    },
     "edition_key": {
```

Comment:
Can we move this as well? I know it's not being used by anything yet, but it just looks so lonely without its friends...